packages/stream_chat/lib/src/core/models/message.dart

```diff
@@ -72,16 +72,19 @@ class Message extends Equatable {
     this.pinnedBy,
     this.extraData = const {},
     this.deletedAt,
-    this.status = MessageSendingStatus.sent,
+    this.status = MessageSendingStatus.sending,
```

Comment:
Why was this changed? Note that the `fromJson` is setting `MessageSendingStatus.sent`

Reply:
because we need that in order to have the right behaviour in the message input
a new created message has a sending status, while if you get it from backend we set it as send